src/test/scala/no/gdl/bookapi/model/domain/TranslationTest.scala

```diff
@@ -69,6 +69,70 @@ class TranslationTest extends IntegrationSuite with TestEnvironment {
     }
   }
 
+  test("that bookIdsWithLanguageAndLevel returns ids with same language when level is None") {
+    withRollback { implicit  session =>
+      val book1 = addBookDef()
+      val book2 = addBookDef()
+      val book3 = addBookDef()
+
+      val translationForBook1 = addTranslationDef("ext1", "title 1", book1.id.get, "xho", None)
```

Comment:
`None` can be omitted because of `readingLevel: Option[String] = None`, or it could be replaced with `readingLevel = None`. More verbose, but easier to read what it refers to, imho.

Reply:
Neither of `translationForBookx` are used, should they be discarded?